base_tier_validation/templates/tier_validation_templates.xml

```diff
@@ -5,80 +5,81 @@
             <button
                 name="request_validation"
                 string="Request Validation"
-                t-attf-attrs="{'invisible': ['|','|',('need_validation', '!=', True),('rejected','=',True),('#{state_field}', '#{state_operator}', #{state_value})]}"
+                t-attf-attrs="{'invisible': ['|', '|',
+                ('need_validation', '!=', True),
+                ('rejected','=',True),('#{state_field}', '#{state_operator}', #{state_value})]}"
                 type="object"
             />
             <button
                 name="restart_validation"
                 string="Restart Validation"
-                t-attf-attrs="{'invisible': ['|',('review_ids', '=', []),('#{state_field}', '#{state_operator}', #{state_value})]}"
+                t-attf-attrs="{'invisible': ['|',
+                ('review_ids', '=', []),
+                ('#{state_field}', '#{state_operator}', #{state_value})]}"
```

Comment:
suggestion: these style changes could be reverted.

Reply:
Helps to read the code better